javascript/javascript1/week1/script.js

```diff
@@ -25,4 +25,43 @@
         console.log (`Your dog will be ${dogYear} human years old in ${dogYearFuture}`);
     }
 
+// Housey pricey (A house price estimator)
+    let detailsOfHouse = [
+        {
+            name: "Peter", 
+            length: 10,
+            breadth: 8,
+            height: 10,  
+            gardenSizeInM2: 100,    
+            houseCost: 2500000,
+        },
+        {
+            name: "Julia", 
+            length: 11,
+            breadth: 5,
+            height: 8,  
+            gardenSizeInM2: 70,    
+            houseCost: 1000000,
+        },
+    ]
+    
+    // Using index number to access details of "Peter" or "Julia"
+    let indexNumber = 0;
```

Comment:
I see the point of this variable , but I don't think it is necessary you could just write `0` instead of using the variable.

Reply:
When the indexNumber = 1, Julie's house details are visible in the console.